source/documentation/administration_guide/topics/Syntax_for_the_engine-backup_Command.adoc

```diff
@@ -13,29 +13,32 @@ The `engine-backup` command works in one of two basic modes:
+*`--file`*:: Specifies the path and name of a file (for example, _file_name_.backup) into which backups are saved in backup mode, and to be read as backup data in restore mode. The path is defined by default as `/var/lib/ovirt-engine-backup/`.
 
-*`--log`*:: Specifies the path and name of a file into which logs of the backup or restore operation are to be written. This parameter is required in both backup mode and restore mode.
+*`--log`*:: Specifies the path and name of a file ( for example _log_file_name_) into which logs of the backup or restore operation are written. The path is defined by default as `/var/log/ovirt-engine-backup/` .
 
-*`--scope`*:: Specifies the scope of the backup or restore operation. There are four options: `all`, which backs up or restores all databases and configuration data; `files`, which backs up or restores only files on the system; `db`, which backs up or restores only the {engine-name} database; and `dwhdb`, which backs up or restores only the Data Warehouse database. The default scope is `all`.
+*`--scope`*:: Specifies the scope of the backup or restore operation. There are four options: `all`, to back up or restore all databases and configuration data (set by default); `files`, to back up or restore only files on the system; `db`, to back up or restore only the {engine-name} database; and `dwhdb`, to back up or restore only the Data Warehouse database.
 +
-The `--scope` parameter can be specified multiple times in the same `engine-backup` command.
+The `--scope` option can be specified multiple times in the same `engine-backup` command.
 
 *{engine-name} Database Options*
 
 The following options are only available when using the `engine-backup` command in `restore` mode. The option syntax below applies to restoring the {engine-name} database. The same options exist for restoring the Data Warehouse database. See `engine-backup --help` for the Data Warehouse option syntax.
 
-*`--provision-db`*:: Creates a PostgreSQL database for the {engine-name} database backup to be restored to. This is a required parameter when restoring a backup on a remote host or fresh installation that does not have a PostgreSQL database already configured.
+*`--provision-db`*:: Creates a PostgreSQL database for the {engine-name} database backup to be restored to. This is a required option when restoring a backup on a remote host or fresh installation that does not have a PostgreSQL database already configured. When this option is used in restore mode, the `--restore-permissions` option is added by default.
 
-*`--change-db-credentials`*:: Allows you to specify alternate credentials for restoring the {engine-name} database using credentials other than those stored in the backup itself. See `engine-backup --help` for the additional parameters required by this parameter.
+*`--provision-all-databases`*:: Creates databases for all data dumps included in the archive. When enabled, this is the default.
```

Comment:
Consider:
"dump" -- as the IBM guide says, "Use this term with caution to avoid potentially offensive phrasing such as "take a dump". Preferred usage includes the modifier "memory" (for example, "When the program ends abnormally, it creates a memory dump.")." Obviously, up to you.